sdk/build.gradle

```diff
@@ -0,0 +1,50 @@
+buildscript {
```

Comment:
This gradle file shouldn't use FG at all if this is for a plugin for SpongeAPI...